### ISSUE TITLE
IHub does not inherit methods

### ISSUE THREAD

**Issue by szingg** [nenoNaninu/TypedSignalR.Client#75]
First of all, thanks for your awesome work.

The following usecase does not work:
``` csharp
interface IHub : IAnyInterface {}

interface IAnyInterface
{
    Task<Guid> HubMethod2(string message, int value);
}

var hub = hubConnection.CreateHubProxy<IHub>();
```

build output:
```
Error	CS0535	'Extensions.HubInvokerForXyServicesIHub' does not implement interface member 'IAnyInterface.HubMethod2(string, int)'	<path>\TypedSignalR.Client\TypedSignalR.Client.SourceGenerator.ExtensionsSourceGenerator\TypedSignalR.Client.Extensions.Internal.Generated.cs
```

**Comment by nenoNaninu** [comment]
Thank you for the report! I did not care about such a use case.

I want to support that use case in the future.

**Comment by christallire** [comment]
I need this :p

**Comment by nenoNaninu** [comment]
I released `v3.4.0`. This version supports interface inheritance.
